.cbmc-batch/jobs/aws_cryptosdk_md_init/README.md

```diff
@@ -0,0 +1,3 @@
+# Memory safety proof for aws_cryptosdk_md_init
+
+This proof harness attains 100% code coverage.
```

Comment:
See previous comment. You can remove this README file.

Reply:
Done!